jmeter-service/main.go

```diff
@@ -196,59 +199,47 @@ func getGatewayFromConfigmap() (string, error) {
 }
 
 func sendTestsFinishedEvent(shkeptncontext string, incomingEvent cloudevents.Event, startedAt time.Time, logger *keptnutils.Logger) error {
-
 	source, _ := url.Parse("jmeter-service")
 	contentType := "application/json"
 
-	var testFinishedData interface{}
+	testFinishedData := keptnevents.TestsFinishedEventData{}
+	// fill in data from incoming event (e.g., project, service, stage, teststrategy, deploymentstrategy)
 	if err := incomingEvent.DataAs(&testFinishedData); err != nil {
 		logger.Error(fmt.Sprintf("Got Data Error: %s", err.Error()))
 		return err
 	}
-	testFinishedData.(map[string]interface{})["startedat"] = startedAt
+	// fill in timestamps
+	testFinishedData.Start = startedAt.Format(time.RFC3339)
+	testFinishedData.End = time.Now().Format(time.RFC3339)
 
 	event := cloudevents.Event{
 		Context: cloudevents.EventContextV02{
 			ID:          uuid.New().String(),
 			Time:        &types.Timestamp{Time: time.Now()},
-			Type:        "sh.keptn.events.tests-finished",
+			Type:        "sh.keptn.event.tests.finished",
```

Comment:
Do we have this type in go-utils? If so, can/should we use it?

Reply:
Thanks, good catch. Fixed.